### PR TITLE
feat: add views and initialView configuration

### LINKED ISSUE #5: Hide calendar grid
https://wordpress.org/support/topic/only-show-list-view/#post-15750329

Possibly could have a full `views: {}` override option for maximum flexiblity.

### PR DESCRIPTION
**Pull request by mwguerra**
- all current default settings were kept
- add 3 new shortcode parameters (views, initial_view and enforce_listview_on_mobile)
- allow to choose view types and the initialView
- all view types are shown as buttons
- allow title and today buttons to be hidden
- change the list view on mobile to optional

Resolves #5

### CODE REVIEW

**Comment by lbell** [comment]
Howdy! Thanks for the contributions. Not sure why GitHub didn't ping me about this. I'll hopefully have a chance to take a look here shortly.

**Comment by mwguerra** [comment]
Happy to contribute! Thank you!

File: public/js/pgcal.js [hunk 1]
```diff
@@ -71,20 +82,24 @@ document.addEventListener("DOMContentLoaded", function () {
...
+        window.innerWidth !== width &&
+        views.hasList &&
+        views.wantsToEnforceListviewOnMobile
+      ) {
         if (pgcal_is_mobile()) {
-          calendar.changeView(listType);
-        } else {
-          calendar.changeView("dayGridMonth");
+          selectedView = calendar.view.type
+          return calendar.changeView(views.listType);
```
Comment (lbell) [inline public/js/pgcal.js:96]: This was misbehaving as a return, so I set it back to a straight `calendar.changeView(views.listType);` Could have been something else I was tweaking, but is working now. 

What was the reason for adding the `return`?

Comment (mwguerra) [reply public/js/pgcal.js:96]: No reason at all. It passed my tests here (but it shouldn't - just saw it's type explicitly returns void). I usually don't like else statements on my methods, so I return the main value/behavior at the end and return alternative values before, on each condition. I find it more readable.

**Pull request merged**